src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -21,21 +21,54 @@ public class StringUtil {
      *       containsWordIgnoreCase("ABc def", "AB") == false //not a full word match
      *       </pre>
      * @param sentence cannot be null
-     * @param word cannot be null, cannot be empty, must be a single word
+     * @param phrase cannot be null, cannot be empty, must be a single word
      */
-    public static boolean containsWordIgnoreCase(String sentence, String word) {
+    public static boolean containsWordIgnoreCase(String sentence, String phrase) {
```

Comment:
is there a reason you changed the implementation of the existing `containsWordIgnoreCase`, rather than creating a new method `containsPhraseIgnoreCase`?

Reply:
Since breed allows for spaces in words (eg golden retriever), i updated this method to check for matching phrases. By creating a new method, i would have to use 2 methods to check for matching attributes, which requires me to check if the keyword being checked is 1 word or more than 1 word.